packages/core/src/common/keybinding.spec.ts

```diff
@@ -73,31 +75,31 @@ describe('keybindings', () => {
     it("should set a keymap", () => {
         const rawKeybindings: RawKeybinding[] = [{
             command: "test.command",
-            keybinding: "ctrl+c"
+            keybinding: "shift+c"
```

Comment:
I didn't remember why these needed to be changed to "shift" but after our discussion, we established that it can be changed back to ctrl since issue was fixed in regards to not mapping this to meta on macOS.

Reply:
I used sinon to stub the os verification at the beginning of the tests, so that we don't need to change the `ctrl` to `shift` anymore. Can you confirm that the tests still pass on your machine @MarkZ3 ?